rst2pdf/tests/conftest.py

```diff
@@ -24,6 +24,23 @@
 REFERENCE_DIR = os.path.join(ROOT_DIR, 'reference')
 
 
+def has_plantuml():
```

Comment:
Is there a way we can do this without needing a new function for every dependency we want to add?

Reply:
I did that to give flexibility on how to check if a dependency is satisfied, but yes, it would be better to also have a function to handle the common case of just running a command. I have an idea for that using inner functions. I will push it here so we can review it.